torchvision/prototype/features/_feature.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
What's the minimum Python version that supports this?

Reply:
Probably, 3.7 (https://docs.python.org/3/whatsnew/3.7.html#pep-563-postponed-evaluation-of-annotations)
By the way, we are already doing that in `_image.py`, `_label.py` and others